apps/web/src/pages/templates/components/templates-store/TemplatesStoreModal.tsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable max-len */
+import { useState } from 'react';
+import { ActionIcon, Modal, useMantineTheme } from '@mantine/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconName } from '@fortawesome/fontawesome-svg-core';
+
+import { Button, colors, shadows } from '../../../../design-system';
+import { Close } from '../../../../design-system/icons/actions/Close';
+import {
+  CanvasHolder,
+  GroupName,
+  MadeByNovuStyled,
+  ModalBodyHolder,
+  NovuButtonHolder,
+  TemplateDetails,
+  TemplateHeader,
+  TemplateItem,
+  TemplateName,
+  TemplatesDetailsHolder,
+  TemplatesGroup,
+  TemplatesSidebarHolder,
+  TemplateDescription,
+  useStyles,
+} from './templateStoreStyles';
+
+const TEMPLATES_GROUPED = [
+  {
+    name: 'Collaboration',
+    templates: [
+      {
+        name: ':fa-regular fa-message: Comments',
```

Comment:
This is how we agreed on how the icons will be assigned to the template. Below there is a code that extracts that information.